src/components/Reports/ReportsHoursVisits/ReportsHoursVisits.js

```diff
@@ -27,51 +28,69 @@ const FormatWeekDayIndex = ({ value, format }) => {
   );
 };
 
+const getProcessedVisits = (hoursVisitsdata) => {
+  const processedVisits = hoursVisitsdata.value.reduce(
+    (accumulator, item) => {
+      accumulator.byWeekDayAndHour[item.weekday][item.hour].qVisitors += item.qVisitors;
+      if (item.qVisitorsWithEmail || item.qVisitorsWithEmail === 0) {
+        accumulator.byWeekDayAndHour[item.weekday][item.hour].qVisitorsWithEmail +=
+          item.qVisitorsWithEmail;
+        accumulator.byWeekDayAndHour[item.weekday][item.hour].qVisitorsWithOutEmail +=
+          item.qVisitorsWithOutEmail;
+      }
+      if (item.qVisitors > accumulator.max) {
+        accumulator.max = item.qVisitors;
+      }
+      return accumulator;
+    },
+    { byWeekDayAndHour: createEmptyWeekDayHoursMatrix(), max: 0 },
+  );
+
+  return processedVisits;
+};
+
+const fakeData = getProcessedVisits({
+  success: true,
+  value: prepareFakeVisitsWeekdayHoursData(),
+});
```

Comment:
This is the placeholder data right? For loading only?

Reply:
Yes. it is ...